app/components/steps_component/steps_component.scss

```diff
@@ -5,17 +5,15 @@
   margin-bottom: govuk-spacing(6);
   margin-top: govuk-spacing(6);
 
-  @include govuk-media-query($from: tablet) {
-    width: max-content;
-  }
-
   @include govuk-media-query($from: desktop) {
     width: unset;
   }
 
   h2 {
     margin-bottom: govuk-spacing(2);
     margin-top: govuk-spacing(4);
+
+
   }
```

Comment:
extra lines

Reply:
was literally writing the same. sass lint should have picked up on this, ill investigate why sepeartly

Reply:
Whoopsie